### DISCUSSION TITLE
Can I have one banner configured to appear at the top of some screen and another one to appear at the bottom of other screen?

### DISCUSSION THREAD

**Discussion by yosoychiro** [Poing-Studios/godot-admob-editor#108] [Q&A] [answered]
Hello there! How are you? 
I created two different banner ads, one is supposed to appear on TOP of some screen, and the other one at the BOTTOM of the pause screen.
I noticed the "Banner on top" check affects both banners. Is it possible to achieve this?

![image](https://github.com/Poing-Studios/godot-admob-android/assets/12460817/24d2cf46-f7b5-45ca-9f97-3c67f92f71a4)

**Comment by gumaciel** [comment, marked as answer]
To change the position of the banner, you need to change it before you load the new banner.

```gdscript
MobileAds.config.banner.position = 0 (bottom)
MobileAds.config.banner.position = 1 (top)
```

like this:

https://github.com/Poing-Studios/godot-admob-plugin/blob/v2/addons/admob/test/Example.gd#L262-L265


Config variable:

https://github.com/Poing-Studios/godot-admob-plugin/blob/v2/addons/admob/src/utils/AdMobSettings.gd#L12

**Reply by yosoychiro** [reply]
Thanks a lot!!!
